cykly/cyklus-for.md

```diff
@@ -26,7 +26,7 @@ Nikdo nás samozřejmě nenutí procházet pouze pole čísel. Mějme napříkla
 
 ```js
 const jmena = ['petr', 'pavel', 'jitka', 'jana'];
-for (const jmeno of jmena) {
+for (const jmeno in jmena) {
```

Comment:
@podlomar, `of` bylo správně, ne?

![image](https://github.com/Czechitas-podklady-WEB/uvod-do-js/assets/1045362/da426cc8-853a-48d3-9f9c-20122ac3459e)